### PR TITLE
Allow for specifying kubeconfig-name in the config file

### PR DESCRIPTION
**Pull request by herbygillot**

### CODE REVIEW

**Comment by danielfoehrKn** [comment]
Thanks, I ll try to take a look at it today!

File: cmd/switcher/switcher.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ import (
 
 const vaultTokenFileName = ".vault-token"
 
+const defaultKubeconfigName = "config"
```
Comment (danielfoehrKn) [inline cmd/switcher/switcher.go:40]:
```suggestion
const (
	vaultTokenFileName = ".vault-token"
	defaultKubeconfigName = "config"
)
```

**Review by danielfoehrKn — CHANGES_REQUESTED** [review]
Besides a nit I tested it and it works. Thanks!

I already had in mind to make this configurable per path, but this is also great as a global default.

**Comment by herbygillot** [comment]
> Besides a nit I tested it and it works. Thanks!
> 
> I already had in mind to make this configurable per path, but this is also great as a global default.

Resolved. I 100% agree with you that this should also be available per-path, but `kubeswitch` definitely needed this as a core config option just to start.

**Review by danielfoehrKn — APPROVED** [review]

**Comment by herbygillot** [comment]
BTW @danielfoehrKn, can we cut a minor release with this functionality?  Maybe an 0.3.1...? Thanks.

**Comment by danielfoehrKn** [comment]
Sure, will create a patch release with `0.3.1 `

**Comment by danielfoehrKn** [comment]
Release[ job was successful](https://github.com/danielfoehrKn/kubeswitch/actions/runs/620423047) and the release is available via homebrew.

**Comment by herbygillot** [comment]
Excellent, thank you.  The [port has been updated](https://github.com/macports/macports-ports/commit/3e1210d2da57478311406006c593985aa95c9c20), so this release will also be available in MacPorts shortly as well.

**Pull request merged**